crates/cubecl-linalg/src/matmul/components/global/args.rs

```diff
@@ -1,5 +1,6 @@
+use crate::tensor::{VirtualTensorOperations, VirtualTensorOperationsExpand};
 use cubecl::prelude::*;
-use cubecl_core as cubecl;
+use cubecl_core::{self as cubecl};
 
 #[cube]
```

Comment:
Now that we have the new VirtualTensor trait, it looks like we can replace this with a struct. Something like 

```rust
pub struct MatmulArgs<L, R, O> 
where 
   L: VirtualTensor,
   R: VirtualTensor,
   O: VirtualTensor
{
   lhs: L,
   rhs: R,
   out: O
}
```

This seems even more relevant as there is only a single implementation of `MatmulArgs`